cmd/esmapping-generator/app/flags.go

```diff
@@ -15,28 +15,34 @@
 package app
 
 import (
+	"github.com/jaegertracing/jaeger/pkg/es/config"
 	"github.com/spf13/cobra"
 )
 
 // Options represent configurable parameters for jaeger-esmapping-generator
 type Options struct {
 	Mapping       string
 	EsVersion     uint
-	Shards        int64
-	Replicas      int64
+	Indices       config.Indices
 	IndexPrefix   string
 	UseILM        string // using string as util is being used in python and using bool leads to type issues.
 	ILMPolicyName string
 }
 
 const (
-	mappingFlag       = "mapping"
-	esVersionFlag     = "es-version"
-	shardsFlag        = "shards"
-	replicasFlag      = "replicas"
-	indexPrefixFlag   = "index-prefix"
-	useILMFlag        = "use-ilm"
-	ilmPolicyNameFlag = "ilm-policy-name"
+	mappingFlag              = "mapping"
+	esVersionFlag            = "es-version"
+	spanShardsFlag           = "shards-span"
```

Comment:
this is repeating logic from cmd/es-rollover/app/init/flags.go, can we create a helper function and reuse?